airbyte-integrations/base-python/airbyte_protocol/airbyte_protocol/airbyte_message.py

```diff
@@ -0,0 +1,57 @@
+# generated by datamodel-codegen:
+#   filename:  airbyte_message.yaml
+
+from __future__ import annotations
+
+from enum import Enum
+from typing import Any, Dict, Optional
+
+from pydantic import BaseModel, Field
+
+
+class Type(Enum):
```

Comment:
nice! these look great.

Reply:
This being said, should the generated classes be checked in?

Reply:
I think it's really confusing if they aren't checked in, especially since it won't work until you run the build. I think we should also check in the Java json-schema generated types as well. 

Without the git diff it makes it difficult to know what is being used.